.github/workflows/update-github.yaml

```
@@ -0,0 +1,45 @@
+name: "UpdateOctokit"
+
+on:
+  schedule:
+    - cron: '0 18 * * 0'
+
+jobs:
+  UpdateOctokit:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Update Octokit
+      working-directory: packages/github 
+      run: |
+        npx npm-check-updates -u --dep prod
+        npm install
+    - name: Check Status
+      id: status
+      working-directory: packages/github 
+      run: | 
+        if [[ "$(git status --porcelain)" != "" ]]; then 
+        echo "::set-output name=createPR::true"
+        git config --global user.email "bot@github.com"
+        git config --global user.name "Dependency Update Bot"
```

```suggestion
        git config --global user.email "github-actions[bot]@users.noreply.github.com"
        git config --global user.name "github-actions[bot]"
```

I am using this way in [peaceiris/actions-gh-pages: GitHub Actions for GitHub Pages](https://github.com/peaceiris/actions-gh-pages).